debian/usr.bin.snap-confine

```diff
@@ -112,6 +114,11 @@
     mount options=(rw bind) /snap/ubuntu-core/*/libx32/ -> /libx32/,
     mount options=(rw bind) /snap/ubuntu-core/*/lib64/ -> /lib64/,
     mount options=(rw bind) /snap/ubuntu-core/*/usr/ -> /usr/,
+    mount options=(rw bind) /snap/ubuntu-core/*/etc/alternatives/ -> /etc/alternatives/,
+
+    # Allow snaps to share content amongst themselves.
```

Comment:
Please adjust this to be:
\# Allow snaps to share content via the 'content' interface